### PR TITLE
[REVIEW] Add queries logger to Physical plan

### LINKED ISSUE #1306: Revamping Loggers: queries_logger print physical plan
Parent https://github.com/BlazingDB/blazingsql/issues/1297

### PR DESCRIPTION
**Pull request by diegodfrf**
Fixes #1306
Related to #1297

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: engine/src/CalciteInterpreter.cpp [hunk 1]
```diff
@@ -31,14 +32,22 @@ std::shared_ptr<ral::cache::graph> generate_graph(std::vector<ral::io::data_load
...
+		if(batchLogger){
+            batchLogger->info("{query_id}|{step}|{substep}|{info}|||||",
+                              "query_id"_a=queryContext.getContextToken(),
+                              "step"_a=queryContext.getQueryStep(),
+                              "substep"_a=queryContext.getQuerySubstep(),
+                              "info"_a="\"Query Start\n{}\""_format(tree->to_string()));
 		}
 
+        if(queriesLogger){
+            queriesLogger->info("{query_id}|{step}|{substep}|{info}|||||",
```
Comment (williamBlazing) [inline engine/src/CalciteInterpreter.cpp:44]: The format of the queriesLogger is a little different. Its something like this:

        logger->info("{ral_id}|{query_id}|{start_time}|{plan}|{sql}",
                                        "ral_id"_a=queryContext.getNodeIndex(communicationData.getSelfNode()),
                                        "query_id"_a=queryContext.getContextToken(),
                                        "start_time"_a=eventTimer.start_time(),
                                        "plan"_a=query,
                                        "sql"_a=tree->to_string());

**Review by williamBlazing — CHANGES_REQUESTED** [review]

**Comment by diegodfrf** [comment]
rerun tests

**Comment by williamBlazing** [comment]
rerun-tests

**Pull request merged**